### ISSUE TITLE
Activity stream-based reminders

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/bp-event-organiser#19]
It would be cool to have a feature where admins could opt to remind people of upcoming events. Maybe 24 hours before an event, you want everyone subscribed or in the connected group to get a reminder. Possibly this could work together with bp-activity?

**Comment by r-a-y** [comment]
I wouldn't use the activity stream for this as it will clutter the 
stream.  Email blasts would be better.

Maybe Event Organiser has a pro feature for reminders?  Need to do some 
research.

**Comment by boonebgorges** [comment]
Yeah, good point about cluttering the stream. (Just had a meeting - thus 
all the new tickets ;) )

**Comment by r-a-y** [comment]
I figured as such :)
